CHANGELOG.md

```diff
@@ -4,6 +4,8 @@
 * [#1724](https://github.com/oshi/oshi/pull/1724): Removed deprecated MACOSX value from PlatformEnum and SystemInfo and removed the getCurrentPlatformEnum() method - [@Novaenn](https://github.com/Novaenn).
 * [#1725](https://github.com/oshi/oshi/pull/1725): Removed deprecated process sorting methods from the OperatingSystem class - [@varnaa](https://github.com/varnaa).
 * [#1729](https://github.com/oshi/oshi/pull/1729): Changed the return value of linuxOSPRocess and macOSProcess method getCommandLine() from null-delimited string to space-delimited string - [@prathamgandhi](https://github.com/prathamgandhi).
+* [#1730](https://github.com/oshi/oshi/pull/1730): Changed the return value of getServices() from `OSService[]` to `List<OSService>` in OperatingSystem - [@adrian-kong](https://github.com/adrian-kong).
+* [#1731](https://github.com/oshi/oshi/pull/1731): Added tests for linux `Devicetree`, `Who` classes - [@adrian-kong](https://github.com/adrian-kong).
```

Comment:
This one isn't a breaking change, so in theory it would go in a different section. But also adding tests doesn't really need to be in the change log.